doc/api/child_process.md

```diff
@@ -858,7 +858,7 @@ changes:
     process will be killed. **Default:** `'SIGTERM'`.
   * `maxBuffer` {number} Largest amount of data in bytes allowed on stdout or
     stderr. If exceeded, the child process is terminated. See caveat at
-    [`maxBuffer` and Unicode][]. **Default:** `200 * 1024`.
+    [`maxBuffer` and Unicode][]. **Default:** `Infinity`.
```

Comment:
This change seems wrong. The default is still set to `200 * 1024`. This default was also used in the added tests here.

Reply:
I execute this code. I think that it will throw error if this option is not infinity.
It seems that it is over the default.

```
// maxBuffer size is Infinity at default.
{
  const args = ['-e', "console.log('a'.repeat(200 * 1024 + 1))"];
  const ret = spawnSync(process.execPath, args, { encoding: 'utf-8' });

  assert.ifError(ret.error);
}
```

I should be added this test case.

Reply:
Looks like there is a gap here. Technically, `Infinity` reflects the current state, but should we tighten the validation instead?